apps/rush-lib/src/schemas/command-line.schema.json

```diff
@@ -41,6 +41,11 @@
             "title": "Is Parallelized",
             "description": "Describes whether or not this command can be run in parallel (with multiple instances running on several projects simultaneously). Defaults to 'true'.",
             "type": "boolean"
+          },
+          "optional": {
+            "title": "Is optional and only applies to projects with corresponding npm script",
+            "description": "Indicates whether the command runs on every project or just projects that have the corresponding npm script.",
```

Comment:
>"Indicates whether the command runs on every project or just projects that have the corresponding npm script." [](start = 27, length = 110)

"Normally Rush requires that each project's package.json has a \"scripts\" entry corresponding to the command name, to avoid oversights when new projects are added to the system. Set ignoreMissingScript to false to disable this check."